client/src/AppBundle/Service/Client/Internal/ClientApi.php

```diff
@@ -0,0 +1,79 @@
+<?php
```

Comment:
A slight pet peeve of mine - could we get a `declare(strict_types=1)` at the start of new files? It just means we get type errors thrown when we try to provide the incorrect type to functions.